src/components/Search/Search.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { MoviesList } from '../MoviesList';
+
+export class Search extends Component {
+  state = {
+    query: '',
+  };
+
+  render() {
+    const { movies } = this.props;
+    const textToLower = this.state.query.toLocaleLowerCase();
+    const visibleMovies = [...movies].filter(({ title, description }) => (
```

Comment:
Filter method returns new array, so you don't need to create a copy of movies.

Reply:
you are right